test/compat/coffee-for-loops.civet

```diff
@@ -247,12 +245,9 @@ describe "coffeeForLoops", ->
     "civet coffeeCompat"
     coffees = (s for s in scripts when s.type in coffeetypes)
     ---
-    var coffees, s
-    var indexOf: <T>(this: T[], searchElement: T) => boolean = [].indexOf as any;
+    var coffees, s;var indexOf: <T>(this: T[], searchElement: T) => boolean = [].indexOf as any;
```

Comment:
The lack of newline between the two var statements here is sad, but not terrible.